httpbin/cmd/cmd.go

```diff
@@ -233,6 +241,12 @@ func loadConfig(args []string, getEnv func(string) string, getHostname func() (s
 			return nil, configErr("https cert and key must both be provided")
 		}
 	}
+	if cfg.LogFormat == "" && getEnv("LOG_FORMAT") != "" {
```

Comment:
This doesn't work as intended, because `cfg.LogFormat` will always have a default value of `"text"` based on the way the flag is defined above.

I think the best we can do is handle this like we do other config options with default values, where we check the env var when the value matches the default:
https://github.com/mccutchen/go-httpbin/blob/874932b8a03a74758ac8ef61439fcd4a0550211f/httpbin/cmd/cmd.go#L193-L201

It's not perfect, but it's good enough.

Reply:
Ohh I see definitely, sorry for logic mistake. Will fix that as well!